### PR TITLE
Document fixes, corrections, and wording update

### PR DESCRIPTION
**Pull request by jadeddelta**
This PR contains updates to documentation and other descriptive files, fixing:

- Fixing various broken demos 
- Fixing the contributions link in the code of conduct page
- Grammar issues/maintaining a consistent style between similar elements 
- _Establishing a consistent term for participants in a subject_

The last point involves changing from subject -> participant in order to standardize terminology. Participant is favored over subject due to subject having a perceived [lack of agency/choice](https://blog.primr.org/subjects-vs-participants/) in an experiment, and also due to the word subject shifting towards referring to non-human participants in an experiment. For this point, the files changed included:
- Most documentation
- Most demos (including comments) 
- Plugin description in a package.json

Files that have not been changed include:
- Core code/commenting 
- Documentation and demos that specifically refer to subject IDs (Ex. MTurk)

For later (8.0?), it would be a good idea to phase out subject entirely to keep consistency, but it's not a big big priority. 
also, added self to contributors.md :)

### CODE REVIEW

**Comment by jadeddelta** [comment]
i think i know what’s wrong, ill fix this up when i get the chance, something went wrong when i tried to fix a merge conflict

**Comment by becky-gilbert** [comment]
Thanks for this contribution @jadeddelta! ❤️ 

If it's ok with you and @jodeleeuw, I'd like to review and merge this tomorrow or Friday. I'd also like to handle some other documentation issues and PRs at the same time (#2728, #2548, #2495, #2400) and update the live docs.

**Comment by jodeleeuw** [comment]
Thanks @becky-gilbert that sounds great 👍😃

**Comment by jadeddelta** [comment]
sounds like a plan! if there is anything i can do to help let me know. :)

File: docs/plugins/list-of-plugins.md [hunk 1]
```diff
@@ -8,55 +8,55 @@ For an overview of what plugins are and how they work, see our [plugins overview
...
-[external&#8209;html](external-html.md) | Displays an external HTML page (such as a consent form) and lets the subject respond by clicking a button or pressing a key. Plugin can validate their response, which is useful for making sure that a subject has granted consent before starting the experiment.
-[free&#8209;sort](free-sort.md) | Displays a set of images on the screen in random locations. Subjects can click and drag the images to move them around the screen. Records all the moves made by the subject, so the sequence of moves can be recovered from the data.
+[external&#8209;html](external-html.md) | Displays an external HTML page (such as a consent form) and lets the participant respond by clicking a button or pressing a key. Plugin can validate their response, which is useful for making sure that a participant has granted consent before starting the experiment.
+[free&#8209;sort](free-sort.md) | Displays a set of images on the screen in random locations. participants can click and drag the images to move them around the screen. Records all the moves made by the participant, so the sequence of moves can be recovered from the data.
```
Comment (becky-gilbert) [inline docs/plugins/list-of-plugins.md:25]: Capital P here too.

**Review by becky-gilbert — APPROVED** [review]
Thanks again @jadeddelta! This is fantastic 🤩 
BTW you can ignore my comments - just a few minor change that I'll make after merging.

**Comment by jodeleeuw** [comment]
Hiya @becky-gilbert. I'm guessing you are still working on the rest of these docs updates? I didn't see a new deploy of the docs yet. LMK if you need something from me!

**Comment by becky-gilbert** [comment]
Yes, sorry, still working on it which is why I haven't deployed yet! 

Also I'm a little nervous because I haven't updated the docs since you switched to versioned docs. But since there's no new release with these updates I think it should be straightforward right? In any case I'll be sure to check everything locally before deploying.

**Comment by becky-gilbert** [comment]
FYI, these changes are now live! 🚀

**Pull request merged**